tutorials/plot_background_filtering.py

```diff
@@ -894,6 +894,102 @@ def baseline_plot(x):
+# --------------------------------------------------------
+# In the :ref:`tut_filtering_in_python` section, we have already talked about
+# the default filter lengths and transition bandwidths that are used when no
+# custom values are specified using the respective filter function's arguments.
+#
+# If you want to find out about the filter length and transition bandwidth that
+# were used through the 'auto' setting, you can use `mne.filter.create_filter`
+# to print out the settings once more:
+
+# Use the same settings as when calling e.g., `raw.filter()`
+fir_coefs = mne.filter.create_filter(data=None,  # Data is only used for sanity checking, not strictly needed  # noqa
+                                     sfreq=1000.,  # sfreq of your data in Hz
+                                     l_freq=None,
+                                     h_freq=40.,  # assuming a lowpass of 40 Hz
+                                     method='fir',
+                                     fir_window='hamming',
+                                     fir_design='firwin')
+
+# See the printed log for the transition bandwidth and filter length
+# Alternatively, get the filter length through:
+filter_length = fir_coefs.shape[0]
+
+#
+# .. note:: If you are using an IIR filter, `mne.filter.create_filter` will not
+#           print a filter length and transition bandwidth to the log. Instead,
+#           you can specify the roll off with the `iir_params` argument or stay
+#           with the default, which is a 4th order (Butterworth) filter.
+#
+#
+# Passband ripple and stopband attenuation
+# ----------------------------------------
+#
+# When use standard `scipy.signal.firwin` design (as for fir filters in MNE),
+# the passband ripple and stopband attenuation are dependent upon the window
+# used in design. For standard windows the values are listed in this table (see
+# Ifeachor & Jervis, p. 357 [3]_):
+#
+# +-------------------------+-----------------+----------------------+
+# | Name of window function | Passband ripple | Stopband attenuation |
+# +=========================+=================+======================+
+# | Hann                    | 0.0545 dB       | 44 dB                |
+# +-------------------------+-----------------+----------------------+
+# | Hamming                 | 0.0194 dB       | 53 dB                |
+# +-------------------------+-----------------+----------------------+
+# | Blackman                | 0.0017 dB       | 74 dB                |
+# +-------------------------+-----------------+----------------------+
+#
+#
+# Filter delay and direction of computation
+# -----------------------------------------
+# For reporting this information, it might be sufficient to read the docstring
+# of the filter function or method that you apply. For example in the
+# docstring of `mne.filter.create_filter`, for the phase parameter it says:
+#
+#    "Phase of the filter, only used if ``method='fir'``.
```

Comment:
no real need for leading and trailing quotes, this will already render as a block quote so we can let the automatically inserted left bar `|` imply the quoting